Dockerfile

```diff
@@ -72,14 +72,27 @@ RUN mkdir -p ./vendor-assets && \
     find public/ -regex "public\/${regex_path}.*\.${regex_files}" -exec cp --parent "{}" vendor-assets/  \;
 
 
+###
+
+
+FROM node:20 AS assets-build
+
+WORKDIR /code/public/app/themes/justice
```

Comment:
`code/` is a directory we create, using WORKDIR, to isolate the compilation step.

If you take a look in the `docker-compose` node service, you'll see we only take what we need to compile assets - instead of `code`, we call the directory `node`. We could do this here too, maybe?

Reply:
What I mean is, there is no need, other than getting needed files, to recreate the directory structure.

Reply:
Good spot!